fossunited/api/tickets.py

```diff
@@ -0,0 +1,94 @@
+"""
+APIs for Tickets and Transfer Tickets
+"""
+
+import frappe
+
+
+@frappe.whitelist(allow_guest=True)
+def check_ticket_validity(ticket_id: str):
+    """
+    Check if the ticket is valid or not
+    """
+    is_ticket_valid = frappe.db.exists("FOSS Event Ticket", ticket_id)
+    print(is_ticket_valid)
+    if is_ticket_valid:
+        return True
+
+    return False
+
+
+@frappe.whitelist(allow_guest=True)
+def get_ticket_details(ticket_id: str):
+    """
+    Get the event for the ticket
+    """
+    ticket = frappe.db.get_value(
+        "FOSS Event Ticket",
+        ticket_id,
+        ["event", "tier", "wants_tshirt", "tshirt_size"],
+        as_dict=True,
+    )
+    return ticket
+
+
+@frappe.whitelist(allow_guest=True)
+def create_transfer_request(ticket: str, receiver_details: dict):
+    """
+    Create a transfer request for the ticket
+    """
+    print(receiver_details)
+    transfer_request = frappe.get_doc(
+        {
+            "doctype": "FOSS Event Ticket Transfer",
+            "ticket": ticket,
+            "receiver_name": receiver_details.get("receiver_name"),
+            "receiver_email": receiver_details.get("receiver_email"),
+            "designation": receiver_details.get("designation"),
+            "organization": receiver_details.get("organization"),
+            "wants_tshirt": receiver_details.get("wants_tshirt"),
+            "tshirt_size": receiver_details.get("tshirt_size"),
+            "status": "Pending Approval",
+        }
+    )
+    transfer_request.insert(ignore_permissions=True)
+    return transfer_request
+
+
+@frappe.whitelist()
+def check_transfer_doc_validity(transfer_id: str):
```

Comment:
Again, the function name should clarify the intent/what's it doing.

check/validate implies it will be some kind of validate and throw function. This one just returns whether the provided id is valid. 

And according to the document I shared, this should start with `get_`